src/groups/bmq/bmqeval/bmqeval_simpleevaluator.t.cpp

```diff
@@ -520,6 +521,14 @@ static void test3_evaluation()
         {"i_0 != -9223372036854775807", true},  // -(2 ** 63) + 1
         {"i_0 != 9223372036854775807", true},   // 2 ** 63 - 1
         {"i_0 != -9223372036854775808", true},  // -(2 ** 63)
+
+        // exists
```

Comment:
Let's add a comment that `exists` might mean a property or a function depending on its usage:
```suggestion
        // exists
        // Note that we allow to set a message property with name `exists`.
        // In this case the usage context defines whether it's a function call
        // or a property value.
```